base/BUILD.gn

```diff
@@ -2477,7 +2477,7 @@ buildflag_header("debugging_buildflags") {
 
   # TODO(crbug.com/1420774): Try and enable these checks on Android too.
   enable_commandline_sequence_checks =
-      (is_debug || dcheck_always_on) && !is_android
+      (is_debug || dcheck_always_on) && !is_android && !is_cobalt
```

Comment:
Why do we need this? The PR description doesn't explain the origin  
of this comment, why it's needed or why it's OK to skip it.

Reply:
Yes we need this. This was already asked by another reviewer. Please refer to the bug for details.

Reply:
The CL should explain why we need this independently of the bug, because
for example buganizer is not available for external developers.

Reply:
It's the same reason that is already documented in the existing crbug referred to above.